CONTRIBUTING.md

```diff
@@ -39,6 +39,8 @@ Qt:
 * Provide **screenshots** for UI related changes.
 * Keep your git commit history **clean** and **precise**. Commits like
   `xxx fixup` should not appear.
+* Commit message should state not only what has been changed, but also why
```

Comment:
eh, I'm just referring to the last part of the sentence. "what has been changed, but also why a change is needed" sounds better.